css/design_system/components/footer.scss

```diff
@@ -0,0 +1,110 @@
+ *  </div>
+ * </footer>
+ */
+
+.s-footer {
+  @extend .s-p-4;
+  @extend .s-bg-gray-800;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.s-footer-content {
+  * {
+    @extend .s-my-0;
+    @extend .s-text-gray-100;
+    @extend .s-text-start;
+  }
+}
+
+ /**
+ * @title Footer with Links
+ * @category components
+ * @component_type footer
+ * @description Footer with paragraph text and one set of links. Use of the <code>.s-footer-right</code> and <code>.s-footer-link</code> classes ensure proper spacing between links.
+ *
+ * @example
+ * <footer class="s-footer">
+ *  <div class="s-footer-content">
+ *    <p>Turing School of Software and Design, a Colorado Non-Profit Organization.</p>
+ *  </div>
+ *  <div class="s-footer-right">
+ *    <a class="s-footer-link" href="/privacy">Privacy Policy</a>
+ *    <a class="s-footer-link" href="https://turing.edu/programs/" target="blank">Programs</a>
+ *    <a class="s-footer-link" href="https://turing.edu/try-coding/">Try Coding</a>
+ *  </div>
+ * </footer>
+ */
+
+.s-footer-left {
+  > :first-child {
+    @extend .s-ml-0;
+    @extend .s-pl-0;
+  }
+}
+
+.s-footer-right {
+  > :last-child {
+    @extend .s-mr-0;
+    @extend .s-pr-0;
+  }
+}
+
+.s-footer-link {
+  @extend .s-text-uppercase;
+  @extend .s-text-body-sm;
+  @extend .s-font-bold;
+  @extend .s-text-gray-100;
+  @extend .s-pr-4;
+  @extend .s-pl-2;
+  text-decoration: none;
+  letter-spacing: var(--s-space-1);
+  cursor: pointer;
+}
+
+.s-footer-link:hover {
+  @extend .s-text-white;
+}
+
+ /**
+ * @title Footer with Links and Social Icons
+ * @category components
+ * @component_type footer
+ * @description Footer with two set of links; one being social icons. Use of the <code>.s-footer-right</code> and <code>.s-footer-left</code>, and <code>.s-footer-icon</code> and <code>.s-footer-link</code> classes ensure proper spacing between links and icons within their respective containers, as well as vertical alignment.
+ *
+ * @example
+ * <footer class="s-footer">
+ *  <div class="s-footer-left">
+ *    <a class="s-footer-icon" href="https://www.facebook.com/turingschool/"><img src="https://turing-io-assets.s3-us-west-2.amazonaws.com/images/facebook-grey-100.svg"/></a>
+ *    <a class="s-footer-icon" href="https://twitter.com/turingschool"><img src="https://turing-io-assets.s3-us-west-2.amazonaws.com/images/twitter-grey-100.svg"/></a>
+ *    <a class="s-footer-icon" href="https://github.com/turingschool"><img src="https://turing-io-assets.s3-us-west-2.amazonaws.com/images/github-grey-100.svg"/></a>
+ *  </div> 
+ *  <div class="s-footer-right">
+ *    <a class="s-footer-link" href="/privacy">Privacy Policy</a>
+ *    <a class="s-footer-link" href="mailto:robyn@turing.edu">Contact</a>
+ *    <a class="s-footer-link" href="https://writing.turing.edu/" target="blank">Perspectives</a>
+ *  </div>
+ * </footer>
```

Comment:
Same as above - we could condense things down to keep it easier to read and understand what the styles are doing:
```suggestion
 * <footer class="s-footer">
 *  <div class="s-footer-left">
 *    <a class="s-footer-icon"><img src="https://turing-io-assets.s3-us-west-2.amazonaws.com/images/facebook-grey-100.svg"/></a>
 *    <a class="s-footer-icon"><img src="https://turing-io-assets.s3-us-west-2.amazonaws.com/images/twitter-grey-100.svg"/></a>
 *    <a class="s-footer-icon"><img src="https://turing-io-assets.s3-us-west-2.amazonaws.com/images/github-grey-100.svg"/></a>
 *  </div> 
 *  <div class="s-footer-right">
 *    <a class="s-footer-link">Privacy Policy</a>
 *    <a class="s-footer-link">Contact</a>
 *    <a class="s-footer-link">Perspectives</a>
 *  </div>
 * </footer>
```